Foundation/NSXMLNodeOptions.swift

```diff
@@ -46,46 +46,46 @@
 public var NSXMLNodeOptionsNone: Int { return 0 }
 
 // Init
-public var NSXMLNodeIsCDATA: Int { return 1 << 0 }
-public var NSXMLNodeExpandEmptyElement: Int { return 1 << 1 } // <a></a>
-public var NSXMLNodeCompactEmptyElement: Int { return 1 << 2 } // <a/>
-public var NSXMLNodeUseSingleQuotes: Int { return 1 << 3 }
-public var NSXMLNodeUseDoubleQuotes: Int { return 1 << 4 }
-public var NSXMLNodeNeverEscapeContents: Int { return 1 << 5 }
+public var NSXMLNodeIsCDATA: UInt { return 1 << 0 }
```

Comment:
This changes APIs can this be split out to a separate commit? They also present a problem in the regards that UInt does not properly translate to usage patterns in swift since all NSUInteger values are converted to Int not UInt by the importer